README.md

```diff
@@ -31,7 +31,40 @@ Pre-built images using this repository are available on [docker hub](https://hub
     docker run -p 9430:9430 -p 8001:8001 -p 2222:22 -p 8080:8080 -d -P --name=osehravista krmassociates/osehravista-qewd
     ```
 
-## Build Steps
+## Docker Build
+
+### Build Options
+
+#### name_space
```

Comment:
I'd recommend using instance name. This is how the scripts refer to this argument. Namespace in my head is more a Caché term.